src/test/java/org/zalando/nakadi/controller/SubscriptionControllerTest.java

```diff
@@ -176,8 +194,33 @@ public void whenGetNoneExistingSubscriptionThenNotFound() throws Exception {
                 .andExpect(content().string(jsonHelper.matchesObject(expectedProblem)));
     }
 
+    @Test
+    public void whenPostSubscriptionThenWithNoReadScope() throws Exception {
```

Comment:
I think more suitable name will be `whenPostSubscriptionWithNoReadScopeThenForbidden`